### PR TITLE
if start ot goal states are invalid, let recall planner return early

### PR DESCRIPTION
**Pull request by yammineramy**

### CODE REVIEW

File: src/ompl/geometric/planners/experience/src/ThunderRetrieveRepair.cpp [hunk 1]
```diff
@@ -148,6 +148,19 @@ namespace ompl
...
             const base::State *goalState = pis_.nextGoal(ptc);
 
+            if (startState == nullptr) {
+                OMPL_INFORM("RetrieveRepair::solve() start state pointer is null. Returning "
+                            "PlannerStatus::INVALID_START");
+                return base::PlannerStatus::INVALID_START;
+            }
+
+            if (goalState == nullptr)
+            {
```
Comment (dexairob) [inline src/ompl/geometric/planners/experience/src/ThunderRetrieveRepair.cpp:158]: whitespace: this bracket should go on previous line, but not a huge deal.

**Review by dexairob — APPROVED** [review]

File: src/ompl/geometric/planners/experience/src/ThunderRetrieveRepair.cpp (hunk 1 above)
Comment (duanyutong) [reply src/ompl/geometric/planners/experience/src/ThunderRetrieveRepair.cpp]: This repo has a `[.clang-format](https://github.com/DexaiRobotics/ompl/blob/main/.clang-format)` file that is to be used to ensure consistent formatting.

**Pull request merged**